otsserver/stamper.py

```diff
@@ -508,6 +510,18 @@ def is_pending(self, commitment):
             return "Pending confirmation in Bitcoin blockchain"
 
         else:
+            journal = Journal(self.calendar.path + '/journal')
+            idx = self.journal_cursor
+            while idx > 0:
+                # exclude idx==0 meaning stamper loop never executed once
```

Comment:
Would it be better for this path to be `idx==None`?

Reply:
Better indeed
:)